src/utils.jl

```diff
@@ -142,18 +142,16 @@ function reproject!(df::DataFrame, from_crs, to_crs; always_xy = false)
     columns = Tables.columns(df)
     for gc in getgeometrycolumns(df)
         gc in Tables.columnnames(columns) || continue
-        reproject(df[!, gc], from_crs, to_crs; always_xy)
+        df[!, gc] = GeometryVector(reproject(df[!, gc], from_crs, to_crs; always_xy))
     end
     metadata!(df, "crs", to_crs; style = :note)
     metadata!(df, "GEOINTERFACE:crs", to_crs; style = :note)
 end
 
+function reproject(sv::GeometryVector, from_crs, to_crs; always_xy = false)
```

Comment:
This should just work if GeometryVector is a vector.  Should rebuild to a geomvector as well but that may need some more work.  We should talk about GeometryVector separately as well, maybe GeometryOps can own that?